docs/concepts/upgrade-support.md

```diff
@@ -63,8 +63,8 @@ where setting the `upgradeConstraintPolicy` to:
 
```

Comment:
I'm having trouble commenting on the correct line for some reason on Github, but the bullet-pointed list on lines 20-23 looks like it's intended to have multiple levels of indentation. I think the first and fourth bullets can just be removed and the middle two un-indented.

Reply:
Also, line 54 can be changed to "Setting the 'upgradeConstraintPolicy' to:"

Reply:
@trgeiger good catch!

Is [this](https://github.com/operator-framework/operator-controller/compare/77c1ae7943cc4d0ff0de75ccde571a442ff5d4d2..859f2ba7a64cb928332e3506746dc87856b3d7c5) what you had in mind?